openrtx/src/rtx/rtx.cpp

```diff
@@ -126,6 +128,74 @@ void rtx_task()
 
         pthread_mutex_unlock(cfgMutex);
     }
+     /* Spectrum update block, run when in SPECTRUM mode.
+*
+* The spectrum mode is a special mode where the radio is in RX mode but
+* the audio path is disabled. This allows to display the RSSI level of the
+* received signals across a frequency range in a waterfall-like display.
+* 
+* This block writes the received RSSI levels to the spectrum buffer.
+*/
+    #ifdef CONFIG_SPECTRUM
+    if(state.rtxStatus == RTX_SPECTRUM)
+    {
+        //state.spectrum_peakRssi = -160;
+        #define SPECTRUM_WF_LINES 64 // must match NUMBER_BARS in ui_menu.c!
```

Comment:
See above, use a macro in rtx.h: this value is already conflicting with the buffer size set in state.h and, probably, with the one you're using in the UI.